### ISSUE TITLE
Toxicity Experiments

### ISSUE THREAD

**Issue by lesterpjy** [lesterpjy/reFACT#4]
# Initial proposal
- Retain single token adversarial sample generation scheme, investigate how to identify biased circuits related to toxicity.
- Central question: How does bias, such as gender bias and ethnicity bias, reflect in differences of activation of circuits related to toxicity?

# Tasks
- Identify single token words that changes sentence bias, initial idea: ethnicity related surnames @wojciechkosiuk @mazurykm 
- Toxicity prompt templates to tie the single token words with @devin-pe @lesterpjy 
- Familiarizing with TransformerLens library @wojciechkosiuk @mazurykm @devin-pe @lesterpjy 
   - How did the original author calculate difference in prompt completion?
   - What is the "proper" way to compare ground truth logits?
   - Does the single token words (surnames) lead to differences in activation? (ie, does surname lead to bias?)

# Configuration
- Still testing with TransformerLens, so experiments contained as notebooks in the `experiments` directory.
